tdvt/tdvt/tdvt_core.py

```diff
@@ -563,53 +603,105 @@ def get_csv_row_data(tds_name, test_name, test_path, test_result, test_case_inde
         error_msg = test_result.saved_error_message if test_result.saved_error_message else error_msg
         error_type = test_result.get_error_type()
 
-    columns = [suite, test_set_name, tds_name, test_name, test_path, str(passed), str(matched_expected),
-               str(diff_count), test_case_name, test_type, priority, categories, functions, cmd_output,
-               str(error_msg), str(error_type), float(case.execution_time), generated_sql, actual_tuples,
-               expected_tuples]
+    if is_perf_run:
+        columns = [
+            suite,
+            test_set_name,
+            test_name,
+            tds_name,
+            None,
+            None,
+            perf_iteration,
+            None,
+            None,
+            error_msg.replace("\n", "") if error_msg else None,
+            passed,
+            None,
+            None,
+            "Query Time",
+            "TimeTest",
+            case.execution_time
+        ]
+    else:
+        columns = [suite, test_set_name, tds_name, test_name, test_path, str(passed), str(matched_expected),
+                str(diff_count), test_case_name, test_type, priority, categories, functions, cmd_output,
+                str(error_msg), str(error_type), float(case.execution_time), generated_sql, actual_tuples,
+                expected_tuples]
     if test_result.test_config.tested_sql:
         columns.extend([expected_sql, float(expected_time)])
     if test_result.test_config.tested_error:
         columns.extend([actual_error, expected_error])
     return columns
 
-def write_csv_test_output(all_test_results, tds_file, skip_header, output_dir) -> Optional[Tuple[int, int, int, int]]:
+def write_csv_test_output(
+    all_test_results: Dict,
+    tds_file: str,
+    skip_header: bool,
+    output_dir: str,
+    perf_run: bool=False
```

Comment:
Ideally this `perf_run` arg would be a formatter type object where we could (mostly) remove the if/else checks below and encapsulate the different output styles.

Roughly the same feedback for the is_perf_run and tdvt_dialect/perflab dialect